PyroApp/source/Screens/MainScreen/MainScreen.js

```diff
@@ -4,45 +4,93 @@ import {alertsService} from '../../../services/alerts.service';
 import CustomButton from '../../Components/CustomButton';
 import {authService} from '../../../services/auth.service';
 import {useNavigation} from '@react-navigation/native';
+import {LeafletView} from 'react-native-leaflet-view';
+import {SafeAreaView} from 'react-native-safe-area-context';
 
 const MainScreen = () => {
```

Comment:
Looks like you're cheating, the file is a .js one !

Reply:
haha oui @timogrand avait créé des fichiers js